README.md

```diff
@@ -365,3 +365,42 @@ is not initialized yet. Also you may want to stop using Vagrant and you want to
 wipe out all Vagrant control files and resources from your workstation. For those purposes you
 should consider using the `.ci/vagrant-cleaner.sh` script; run `.ci/vagrant-cleaner.sh -h` for
 further information.
+
+
+## Using the kata-local-ci libvirt-based script
+
+Another way to setup a local CI is tu use the `cmd/kata-local-ci` script. This
+script uses `libvirt` in order to setup a virtual machine in which the CI
```

Comment:
You don't need the backticks here as that term is in the spell check dictionary:

- https://github.com/kata-containers/tests/blob/main/cmd/check-spelling/data/projects.txt#L48

Reply:
I wanted the backticks, but I replaced it with a link to the project.